### PR TITLE
updated generateExportEntry to pass node for additional logic downstream

### PR DESCRIPTION
**Pull request by koga73**
exposed the node in generateExportEntry so downstream logic can be written to determine if the node is a class or id

### CODE REVIEW

**Comment by koga73** [comment]
Bump - can someone please review and merge this?
Thank you!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "postcss-modules-scope",
-  "version": "3.0.0",
+  "version": "3.1.0",
```
Comment (alexander-akait) [inline package.json]: Please avoid it in this PR, it will be generate automatically

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Fix the problem above, thank you

**Comment by koga73** [comment]
Complete, reverted package.json version

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**